dot/types/consensus_digest_test.go

```diff
@@ -46,3 +46,25 @@ func TestBabeEncodeAndDecode(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, d, dec)
 }
+
+func TestBabeDecodeVersionedNextConfigData(t *testing.T) {
+	enc := common.MustHexToBytes("0x03010100000000000000040000000000000002")
```

Comment:
Could you point the block number from where this encoded digest was extracted?

Reply:
Done